iothub/device/src/Edge/TrustBundleProvider.cs

```diff
@@ -15,7 +15,7 @@ namespace Microsoft.Azure.Devices.Client.Edge
 {
     internal class TrustBundleProvider : ITrustBundleProvider
     {
-        private static readonly IRetryPolicy s_retryPolicy = new ExponentialBackoffRetryPolicy(3, TimeSpan.FromSeconds(30));
+        private static readonly IIotHubClientRetryPolicy s_retryPolicy = new IotHubClientExponentialBackoffRetryPolicy(3, TimeSpan.FromSeconds(30));
```

Comment:
This is existing code but we shouldn't be reinitializing a retry policy in here. @schoims this could use an update alongside the other retry policy change you're working on for the client.